.github/workflows/test.yml

```diff
@@ -1,12 +1,14 @@
+name: Test
+
 on:
   pull_request:
   push:
     branches:
       - main
 
-name: Test
 jobs:
-  test:
+  go_test:
+    name: testing
     strategy:
       matrix:
         go-version: [ '1.19.x', '1.20.x' ]
```

Comment:
Based on the decision my other PR (#18), you culd also remove the matrix test here and just use Go 1.19 for testing 👍

Reply:
well lets keep it, as 1.21 is about to arrive so we will just change from 1.20 -> 1.21